cmd/daprd/main.go

```diff
@@ -43,7 +45,7 @@ func main() {
 	// set GOMAXPROCS
 	_, _ = maxprocs.Set()
 
-	rt, err := runtime.FromFlags()
+	rt, err := runtime.FromFlags(os.Args[1:])
```

Comment:
I think we can pass in args as is here, in the unlikely scenario that `daprd` is run directly, it will fail right now(master code), but won't it fail with index out of bounds in this code path?